components/brave_sync/extension/background.js

```diff
@@ -234,8 +234,10 @@ class InjectedObject {
         chrome.braveSync.getInitData(arg1/*syncVersion*/);
         break;
       case "sync-setup-error":
-        console.log(`"sync-setup-error" error=${JSON.stringify(arg1)}`);
-        chrome.braveSync.syncSetupError('ERR_SYNC_INIT_FAILED');
+        console.log(`"sync-setup-error" error=${arg1}`);
+        var errorToPass = (arg1 === 'Credential server response 400. Signed request body of the client timestamp is required.') ?
```

Comment:
after changing my time I get the `Credential server response 400` error but not the message after that so it fails under `ERR_SYNC_INIT_FAILED` to me